src/main/java/com/gmail/goosius/siegewar/utils/SiegeWarBattleSessionUtil.java

```diff
@@ -87,8 +87,12 @@ public static void endBattleSession() {
 		 */
 		for (Siege siege : SiegeController.getSieges())
 			endBattleSessionForSiege(siege);
-		
-		Bukkit.getPluginManager().callEvent(new BattleSessionEndedEvent());
+
+		StringBuilder message = new StringBuilder(getBattleSessionEndedMessageHeader(battleResults).toString());
+
+		getBattleSessionEndedMessageLines(battleResults).forEach(translatable -> message.append("\n").append(translatable.toString()));
```

Comment:
I'm not sure what's going on here but it should go back to using Translatables so that this is translatable for all active locales when its broadcast globally.

Reply:
it still uses translatables. the message sent on BattleSessionEndedEvent is built with multiple translatables, the header translatable and other translatables depending on the size of `battleResults`. I moved the logic for building `List<Translatable> lines` and the header translatable into seperate functions so there's not multiple functions running identical code, the send event calls these getter methods and they're also used to construct the message String which is passed to the BattleSessionEndedEvent constructor

i changed it to use defaultLocale() instead of toString() in latest commit tho